workflow/envs/preprocess.yml

```diff
@@ -3,6 +3,7 @@ channels:
   - conda-forge
 dependencies:
   - python=3.11
-  - nd2reader
+  - nd2
+  - xarray
```

Comment:
Do we need to specify xarray as a dependency here? It seems like nd2 uses this (at least I see reference to it later) but if we don't need to import it anywhere I would think we don't need to include it as a dependency ourselves.

Reply:
nd2 will import it as default, but to read the metadata, we need xarray to be installed in this conda env. So yes, we need it here